### ISSUE TITLE
Some countries should get same-culture slave pops

### ISSUE THREAD

**Issue by Idhrendur** [Idhrendur/paradoxGameConverters#147]
https://forum.paradoxplaza.com/forum/index.php?threads/the-eu4-to-v2-converter-project.707362/page-190#post-20750571

**Comment by woggy** [comment]
I'm looking at this with an eye to not copypasting huge chunks from cultureMap into slaveCultureMap. It seems a more sensible approach would be to change what the "default" is in V2World::DetermineDemographics - the things getting african_minor right now are ones that have slaves, but no mapping defined in slaveCultureMap.  I've sketched out a preliminary default-switch based on continent (using the "main" culture in Asia/Oceania), but that is painting things with a fairly wide brush (for one, some granularity between the Middle-Eastern Arabs and, say, Korea, would be nice).

I can't decide whether slaves in African provinces should be african_minor or a relevant local culture (and if the latter, how do we pick which one.)

**Comment by kratostatic** [comment]
On this, I basically agree with what you have done.  I would say that African_minor should be used for Africa for two reasons.  One is that it is simpler, and the other is that from what little I understand of African slavery it tended not to be done to the same groups exclusively so any group of slaves would probably be from different backgrounds.  Although I'm sure that someone with more than superficial knowledge of African history would correct me on this.

What further granularity were you thinking of?  I thought most of the "Arab" slave trade was Berber, who are in Africa anyway.

**Comment by woggy** [comment]
Agreed, african_minor for Africa makes sense.

As for the granularity thing - the entirety of the Middle East is tagged as Asia. I don't actually know enough about the history of the slave trade to know where African slaves ended up, but wherever the dividing line between "use african_minor" and "use local culture" is, it feels like it should be further east than the continental boundary.

**Comment by kratostatic** [comment]
So I've had a think, and I suspect the best thing to do would be to use the continental divide, and add relevant slave cultures (presumably African minor, possibly Afro-Arab although that would have to be created) in the slave culture map for the Arabian cultures (plus Turkish and maybe Persian).

**Comment by woggy** [comment]
Ooh, I like that idea rather a lot. Will work on getting the relevant culture mapping done later today.

**Comment by woggy** [comment]
Augh. have coding done, it works as expected, something has gone *truly* weird trying to update my feature branch. Will try to figure out what's gone wrong with my Git and get a pull request at some point early next week.

Also, I did some research and it looks like the historical African slave trade didn't cover nearly as much of the Middle East as I'd thought; as a result I've tagged all of the Turko-Semitic and Iranian cultures to use african_minor, but only in two of the six EU4 regions. (Probably overkill, I'm fairly certain most of the area will only be a couple of those cultures (Turko-Semitic in particular is a *long* list) at any given point, but better safe than sorry.)
